superset/db_engine_specs/base.py

```diff
@@ -162,21 +242,17 @@ class BaseEngineSpec:  # pylint: disable=too-many-public-methods
 
     # default matching patterns to convert database specific column types to
     # more generic types
-    db_column_types: Dict[utils.GenericDataType, Tuple[Pattern[str], ...]] = {
-        utils.GenericDataType.NUMERIC: (
+    db_column_types: Dict[GenericDataType, Tuple[Pattern[str], ...]] = {
+        GenericDataType.NUMERIC: (
```

Comment:
Do we need this map anymore? I believe this can be achieved with `get_column_spec`